common/configuration/projectconfig_resolver.go

```diff
@@ -143,5 +150,8 @@ func (p ProjectConfigResolver[R]) setMapping(config pc.Config, module optional.O
 		set(&config.Global, mapping)
 	}
 	configPaths := pc.ConfigPaths(p.Config)
+	if len(configPaths) == 0 {
```

Comment:
This is actually unnecessary - `pc.ConfigPaths()` will return `pc.GetDefaultConfigPath()` if `len(p.Config) == 0`

Reply:
Thanks for catching that! https://github.com/TBD54566975/ftl/pull/1276